components/eamxx/src/share/tests/field_utils.cpp

```diff
@@ -127,13 +127,16 @@ TEST_CASE("utils") {
   }
 
   SECTION("horiz_contraction") {
+    // A numerical tolerance
```

Comment:
For the sake of who will maintain this file in the future, you could add a line explaining why we need tol-based tests. Something like
```
// Accumulations in the Kokkos threaded reductions may be done in a different order than
// the manual ones below, so we can only test correctness up to a tolerance
```